pkg/daemon/ceph/osd/volume.go

```diff
@@ -155,6 +157,27 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 			return fmt.Errorf("failed ceph-volume report. %+v", err) // fail return here as validation provided by ceph-volume
 		}
 
+		reportArgs = append(reportArgs, []string{
+			"--format",
+			"json",
+		}...)
+
+		cvOut, err := context.Executor.ExecuteCommandWithOutput(false, "", baseCommand, reportArgs...)
+		if err != nil {
+			return fmt.Errorf("failed ceph-volume json report. %+v", err) // fail return here as validation provided by ceph-volume
+		}
+
+		logger.Debugf("ceph-volume report: %+v", cvOut)
```

Comment:
Do we not get the report in human-readable form anymore? Above on line 131 it will only apply when the metadataDevice is not specified. It would still be useful to always have the output even if we have to call c-v twice.

Reply:
@travisn Yes the human readable report is still there from when it was added in #3149 it is just not changed show it does not show in the diff.  So when metadataDevice is specified we run c-v 3 times and when it is not we run c-v twice.  The json output provides a little more information than the human readable so printing it can be useful for debugging.

Reply:
Ah, now I see it thanks for the clarification